.github/template-files/config.yml

```
@@ -0,0 +1,50 @@
+conda/governance:
+  # [required] community files
+  - CODE_OF_CONDUCT.md
+
+conda/infrastructure:
+  # [required] general workflows
+  - .github/workflows/cla.yml
+  - .github/workflows/update.yml
+
+  # [optional] to include repo in https://github.com/orgs/conda/projects/2
+  - .github/workflows/issues.yml
+  - .github/workflows/labels.yml
+  - .github/workflows/project.yml
```

```suggestion
  # - .github/workflows/issues.yml
  # - .github/workflows/labels.yml
  # - .github/workflows/project.yml
```
Going to comment these ones out first so they don't interfere with our project board automation needs.